docs/encyclopedia/temporal-nexus-security.mdx

```diff
@@ -60,4 +60,18 @@ In Temporal Cloud multiple security provisions are in place to ensure it can act
 ### Payload Encryption and Data Converter {#payload-encryption-data-converter}
 
 The Data Converter works the same for a Nexus Operation as it does for other payloads sent between a Worker and Temporal Cloud.
-There is support for a single Data Converter to be used for inbound inputs and outbound outputs, which means the encryption key often used in a custom Data Converter must be the same for both the caller and the handler.
+The caller and handler Workers must have compatible Data Converters as operation inputs and results are passed between the two.
+
+If encryption keys are used to encrypt payloads, they must be available in both the caller and handler. This may be achieved by either:
+- Using the same encryption key in both the caller and handler, for example a shared symmetric key stored in your KMS.
+- Using different encryption keys and passing the KMS encryption key ID via payload metadata.
```

Comment:
Small nit - we typically try to stay away from Latin words like via. I'd replace via with "through".